### DISCUSSION TITLE
How to use the mdns module of esp32？

### DISCUSSION THREAD

**Discussion by xinruoyusixian** [micropython/micropython#18209] [ESP32]
I saw it in the source code, but I don’t know how to use it

**Comment by peterhinch** [comment]
I suggest you ask in [the forum](http://forum.micropython.org/index.php) which is visited by more users than this bug tracker.

**Comment by xinruoyusixian** [comment]
I tried, but I couldn't find it. I want to set one xxx.local Name of

**Comment by jonnor** [comment]
Does anyone know the answer to this?

**Comment by peterhinch** [comment]
I'll convert this to a discussion where it might find more readers.

**Comment by bixb922** [comment]
See ```network.hostname([name])``` on this page: https://docs.micropython.org/en/latest/library/network.html

This will set the mDNS name on the ESP32.

You must set the hostname before activating/connecting your network interfaces.

The length of the hostname is limited to 32 characters.
